projects/detached_entity/src/remove_sensitive_values_transformer_extensions.rb

```diff
@@ -1,61 +1,63 @@
 module Foobara
-  module RemoveSensitiveValuesTransformerExtensions
-    def from(...)
-      super.tap do
-        create_all_association_types_in_current_namespace(from_type)
+  class DetachedEntity
```

Comment:
Typically we should do `class DetachedEntity < Model` instead here. What you have will work but in cases where behavior is loaded in different orders it can be good to clarify the base class instead of having a situation where we might accidentally create a new class called DetachedEntity that inherits from Object. The downside is if we change the base class we have to change it in multiple places if we specify it in multiple places. That's OK though especially when both places are in the same project.